sdk/core/core/src/az_http_request.c

```diff
@@ -54,39 +54,39 @@ AZ_NODISCARD az_result az_http_request_init(
   return AZ_OK;
 }
 
-AZ_NODISCARD az_result az_http_request_append_path(_az_http_request* p_hrb, az_span path)
+AZ_NODISCARD az_result az_http_request_append_path(_az_http_request* p_request, az_span path)
```

Comment:
`p_request` is better than `p_hrb`, but ideally this instance should be named `ref_request`

Reply:
Yes, I agree